README.md

```diff
@@ -61,57 +63,55 @@ Service to the `c.JupyterHub.services` list:
 ```python
 c.JupyterHub.services = [
     {
-        'name': 'idle-culler',
-        # 'admin': True,
-        'command': [
+        "name": "jupyterhub-idle-culler",
+        "command": [
             sys.executable,
-            '-m', 'jupyterhub_idle_culler',
-            '--timeout=3600'
+            "-m", "jupyterhub_idle_culler",
+            "--timeout=3600",
         ],
+        # "admin": True,
     }
 ]
 ```
 
 where:
 
-- `'admin': True` indicates that the Service requires admin permissions so
-  it can shut down arbitrary user notebooks
-  (only for jupyterhub < 2.0; see [above][permissions]), and
-- `'command'` indicates that the Service will be managed by the Hub.
+- `"command"` indicates that the Service will be managed by the Hub, and
+- `"admin": True` grants admin permissions to this Service and is only meant for
+  use with jupyterhub < 2.0; see [above][permissions].
 
 ### As a standalone script
 
 `jupyterhub-idle-culler` can also be run as a standalone script. It can
 access the hub's api with a service token.
 
-Register the service token with JupyterHub in jupyterhub_config.py:
+Register the service token with JupyterHub in `jupyterhub_config.py`:
 
 ```python
 c.JupyterHub.services = [
     {
-        'name': 'idle-culler',
-        # 'admin': True,
-        'api_token': '...',
+        "name": "jupyterhub-idle-culler",
+        "api_token": "...",
+        # "admin": True,
     }
 ]
 ```
 
 where:
 
-- `'admin': True` indicates that the Service requires admin permissions so
-  it can shut down arbitrary user notebooks
-  (only for jupyterhub < 2.0; see [above][permissions]), and
-- `'api_token'` contains a secret token, e.g. generated by `openssl rand -hex 32`
+- `'api_token'` contains a secret token, e.g. generated by `openssl rand -hex 32`, and
+- `"admin": True` grants admin permissions to this Service and is only meant for
+  use with jupyterhub < 2.0; see [above][permissions].
 
 and store the same token in a `JUPYTERHUB_API_TOKEN` environment variable.
-Then start `jupyterhub-idle-culler` manually
+Then start `jupyterhub-idle-culler` manually.
 
 ```bash
 export JUPYTERHUB_API_TOKEN=api_token_above...
 python3 -m jupyterhub-idle-culler [--timeout=900] [--url=http://localhost:8081/hub/api]
 ```
 
-The command line interface also gives a quick overview of the different options for configuration.
+## Command line flags
```

Comment:
I made this a dedicated section, so it wasn't listed specifically for managed/unmanaged setup.